src/aks-preview/azcli_aks_live_test/configs/cli_matrix_default.json

```diff
@@ -14,7 +14,8 @@
             "test_aks_create_with_monitoring_aad_auth_msi",
             "test_aks_create_with_monitoring_aad_auth_uai",
             "test_aks_enable_monitoring_with_aad_auth_msi",
-            "test_aks_enable_monitoring_with_aad_auth_uai"
+            "test_aks_enable_monitoring_with_aad_auth_uai",
+            "test_aks_nodepool_add_with_ossku_windows2022"
```

Comment:
I suppose this feature won't be added to azure-cli until GA, and the feature would also be removed when it's ready. So, I guess no need for this?

Reply:
Yes, removed it.